daemons/attrd/pacemaker-attrd.c

```diff
@@ -279,7 +279,7 @@ main(int argc, char **argv)
 
         attrd_exit_status = CRM_EX_OK;
         g_set_error(&error, PCMK__EXITC_ERROR, attrd_exit_status, "%s", msg);
-        crm_err(msg);
+        crm_err("%s", msg);
```

Comment:
No idea why this JUST caused a problem. I've run into this type of thing before.

Reply:
Apparently newer compilers are fine with it. I wanted to make sure all my changes still worked on old systems (no longer in CI) and ran into it.